wagtail/wagtailadmin/tests/test_pages_views.py

```diff
@@ -158,6 +158,27 @@ def test_add_subpage_nonexistantparent(self):
     def test_create_simplepage(self):
         response = self.client.get(reverse('wagtailadmin_pages_create', args=('tests', 'simplepage', self.root_page.id)))
         self.assertEqual(response.status_code, 200)
+        self.assertContains(response, '<a href="#content" class="active">Content</a>')
```

Comment:
This ``TestCase`` is getting a bit big... Maybe we should break it down a bit some day.